### DISCUSSION TITLE
Last version which works on Windows XP?

### DISCUSSION THREAD

**Discussion by ghost** [debauchee/barrier#1362] [Q&A] [answered]
And is it compatible with 2.3.3 server?

**Comment by shymega** [comment]
We don't support WIndows XP, and you should be using 2.3.4 at the very least. Try compiling on XP, maybe that might work...

**Reply by ghost** [reply]
This is very SAD you show no respect for retro users. I will have to look up synergy versions or some other software cos this.
Why should I be using 2.3.4 ? :D
2.3.3 was working for me as well but screw this software I need the largest range of compatibility between old a new new oses...

**Reply by shymega** [reply]
EDIT: OK, so I can see now my reply was a little angry/irritable. I'm a bit burned out with OSS work, but that's no excuse for my response. If we had support for Windows XP by developers who can help, as @p12tic says, that'd be good. In terms of CI builds, I haven't found any on AppVeyor, and we use Azure right now, and they don't support XP either. 

We could target a more compatible ABI that works with Windows XP, I suppose. Not sure if that'd work long-term. I think my concern really is that Windows XP is out of date, and whilst it's retro, it is also vulnerable to attacks due to a lack of security updates. 

Unfortunately, things do move on - Qt for example no longer supports XP (latest), so that would mean we might have to stick with older versions of Qt, etc, etc. As Povilas did say, it'd be harder than usual, and a little inconvenient.

~~It's not a matter of respect. It's a matter of our time and CI builds can't target XP, its not supported. Most people don't use XP anymore, and even if they did, it's vulnerable to a wide range of attacks. Believe me, if you were a maintainer, and had to maintain support for a range of old OSes... it would be a bit too much work. 2.3.4 has CRITICAL security fixes, so that's why you should use it. You could compile on XP though. That could work.~~

**Comment by p12tic** [comment, marked as answer]
@feraldruid22 We would be happy if we had developers interested to support Windows XP. It would be a bit harder than usual and a little inconvenient, but certainly doable.

If you really want support for Windows XP the best course of action would be to hire someone who could work on Barrier or support this project financially. At least myself I accept donations for this type of work (see https://www.patreon.com/p12tic), but you can probably find others.
